### DISCUSSION TITLE
Consider adding a "TIP" for new users whom may use Dynamic IP with Cloudflare

### DISCUSSION THREAD

**Discussion by NeoMod** [htpcBeginner/docker-traefik#285] [Ideas]
I would like to suggest mentioning the possibility for new users on Cloudflare to still use this service even if they have a Dynamic IP.

Cloudflare has pretty good documentation, but there is a very nice docker container that can be easily deployed in five minutes: [OZNU Cloudflare DDNS](https://hub.docker.com/r/oznu/cloudflare-ddns)

**Comment by SimpleHomelab** [comment]
I do not know what TIP is. Can you elaborate.

But as for Dynamic IPs. Oznu's cloudflare DDNS image was part of the stake for years, until about last month. I replaced it with qmcgaw/ddns-updater in my recent commit due to the ability to send discord notifications.
